org.eclipse.jdt.core.tests.compiler/src/org/eclipse/jdt/core/tests/compiler/regression/ArrayTest.java

```diff
@@ -505,11 +471,7 @@ public void test016() throws Exception {
 //Check constant pool declaring class of array#clone()
 public void test017() throws Exception {
 	Map optionsMap = getCompilerOptions();
-	CompilerOptions options = new CompilerOptions(optionsMap);
-	if (options.complianceLevel > ClassFileConstants.JDK1_4) {
-		// check that #clone() return type is changed ONLY from -source 1.5 only (independant from compliance level)
-		optionsMap.put(CompilerOptions.OPTION_Source, CompilerOptions.VERSION_1_4);
-	}
+	// check that #clone() return type is changed ONLY from -source 1.5 only (independant from compliance level)
```

Comment:
Please remove the comment // check that ... and the bugzilla reference above the test - they are confusing in the new world.